setup.cfg

```diff
@@ -13,5 +13,8 @@ requires = mantid42, python, python2-numpy, python2-matplotlib, python2-matplotl
 packager = Jean Bilheux <bilheuxjm@ornl.gov>
 
 [flake8]
-exclude = docs,versioneer.py
+statistics = True
+count = True
+exclude = docs,versioneer.py, RefRed/interfaces/refred_main_interface.py, RefRed/interfaces/mplwidget*
 max-line-length = 119
```

Comment:
why not 120? I remember that most of the project set to 120 but not 119.

Reply:
Because it allows for one character at the beginning of the line for a plus or minus sign in a diff when displayed in terminals of 120 columns